deploy/olm-catalog/camel-k/1.0.0-snapshot/camel-k.v1.0.0-snapshot.clusterserviceversion.yaml

```diff
@@ -385,6 +401,14 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resourceNames:
+          - system:image-builder
+          resources:
+          - clusterroles
+          verbs:
+          - bind
```

Comment:
@astefanutti this is auto-generated from the OLM role file. Does this need to be a role or a clusterrole? I'm not sure, since you deal with a cluster resource but you want to bind it locally...

I've left it as you did

Reply:
IIRC, I think a role is enough as it is granted for the operator SA in its namespace.